### PR TITLE
Add_remove 

### PR DESCRIPTION
**Pull request by luqmanmusah**
In this project, I implement the CRUD (create, update, delete) methods. All the elements of the user interface are fully functional.

### CODE REVIEW

File: src/add_remove.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+function clear(tasks) {
+  const temp = [];
+  tasks.forEach((task) => {
+    if (task.completed !== true) {
+      temp.push(task);
+    }
+  });
+  window.update(temp);
+}
```
Comment (bellom) [inline src/add_remove.js]: Oops, let's check the instruction one more time;
image

Let's implement the `clear` function using `filter() method`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @luqmanmusah  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

1. Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luqmanmusah** [comment]
Used the filter method as indicated in the review.

Thanks for the review.

**Review by abruzy — APPROVED** [review]
```markdown
Hi @luqmanmusah 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**